packages/razzle-plugin-scss/README.md

```diff
@@ -1,3 +1,19 @@
 # razzle-plugin-scss
 
-This is just a stub. Not yet implemented.
+This package contains a plugin for using SCSS with Razzle
```

Comment:
We should probably document accepted filename endings (.scss, .sass)?

Reply:
Ah I forgot to put it too. I think it would be nice to put extension in document